config/secrets.yml.dev

```diff
@@ -55,9 +55,14 @@ dev_settings: &DEV_SETTINGS
   asis:
     host: http://localhost:8080
   elasticsearch:
-    reader: localhost
```

Comment:
@nickmarden I agree that this is a minor nit.

I also agree that `elasticsearch:analytics:reader` is more clear than `elasticsearch:reader:analytics`.

I disagree that `analytics:elasticsearch:reader` is more clear than `elasticsearch:analytics:reader`.

Reply:
My thinking about the approach of `analytics:...` is that these ES settings are _currently_ the only information associated with our configuration for analytics and custom indices, but in the future they might not be.

Imagine that later we wanted to add some information about, say, the URL at which we publicly expose the analytics Kibana 5 dashboard. In this case, `kibana5.search.usa.gov` might be a bad name if the Kibana 5 interface for the ASIS/i14y/etc. cluster is a different one than the Kibana5 interface for the analytics (logstash) cluster. So we might want to associate a `dashboard_url` attribute with the analytics portion of our system.

I was thinking that in that case it might be a more obvious organization of attributes to say

```
analytics:
  dashboard_url: https://analytics.search.gov
  elasticsearch:
    reader: ...
    writers:
      - ...
```

than it would be to say

```
elasticsearch:
  analytics:
    dashboard_url: https://analytics.search.gov
    reader: ...
    writers:
      - ...
```

Because the latter implies that the `dashboard_url` is an Elasticsearch setting, whereas the former implies that is a characteristic of our analytics subsystem.

Yes, this is a nit. But since we're reorganizing the app and the corresponding Chef recipes anyway, I thought this was the best time to bring it up.

Reply:
I think we're on the same page then, @nickmarden.  My thinking was that it would only make sense to use `analytics:elasticsearch:reader` if there was something else other than `elasticsearch` that would live underneath `analytics`, and I couldn't think of such a thing that we would ever add there. But you've thought of such a thing: `dashboard_url`.